app/route_rangers_api/models.py

```diff
@@ -135,16 +135,21 @@ class BikeStation(models.Model):
     """
     Class that represent bike sharing docking stations
     """
-
-    station_id = models.CharField(max_length=30, primary_key=True)
+    city = models.CharField(max_length=30, choices=CITIES_CHOICES)
+    station_id = models.CharField(max_length=30)
+    station_name = models.CharField(max_length=30)
+    short_name = models.CharField(max_length=30,null=True)
     location = models.PointField()
 
```

Comment:
looks good to me. Would be curious what `short_name` is/does and why its needed (maybe just for labeling clearly in the frontend?) but otherwise looks good!